exonum/src/events/mod.rs

```diff
@@ -47,13 +48,17 @@ pub enum InternalEvent {
     Timeout(NodeTimeout),
     /// Shutdown the node.
     Shutdown,
+    /// Transaction has been successfully verified.
+    TxVerified(Box<Transaction>),
 }
 
-#[derive(Debug, PartialEq, Eq)]
+#[derive(Debug)]
```

Comment:
Is `PartialEq` and `Eq` vital here? Should it stay removed or should I implement it for `Box<Transaction>`?